sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -1,11 +1,14 @@
 import {
   Body,
   Controller,
+  Get,
+  HttpCode,
+  Patch,
   Post,
   UnprocessableEntityException,
 } from "@nestjs/common";
 import { InstitutionService, UserService } from "../../services";
-import { CreateInstitutionDto } from "./models/institution.dto";
+import { CreateInstitutionDto, InstituteDto } from "./models/institution.dto";
```

Comment:
Please try to keep "Institution" instead of "Institute". It seems to be more accurate.